src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthRpcModule.cs

```diff
@@ -162,7 +162,7 @@ public ResultWrapper<IEnumerable<Address>> eth_accounts()
             return Task.FromResult(GetStateFailureResult<UInt256?>(header));
         }
 
-        Account account = _stateReader.GetAccount(header!.StateRoot!, address);
+        AccountStruct? account = _stateReader.GetAccount(header!.StateRoot!, address);
```

Comment:
Does it have to be nullable? Extra byte it adds is problematic on copy size (128 bytes -> 129 bytes)

better to init with empy code hash and empty storage has and pass that back if null?

Reply:
So been thinking about it, how to avoid copies to. Maybe the answer is something like:

`bool TryGetAccount(root, address, ref AccountStruct account)`

not sure if we could write directly to stack allocated by the calling method?